src/clusterfuzz/_internal/bot/tasks/utasks/symbolize_task.py

```diff
@@ -59,18 +62,30 @@ def utask_main(uworker_input):
 
   # Setup testcase and its dependencies.
   setup_input = setup.preprocess_setup_testcase(testcase)
+  return uworker_io.UworkerInput(
+      job_type=job_type,
+      testcase_id=testcase_id,
+      uworker_env=uworker_env,
+      setup_input=setup_input,
+      testcase=testcase)
+
+
+def utask_main(uworker_input):
+  """Execute the untrusted part of a symbolize command."""
+  job_type = uworker_input.job_type
+  testcase = uworker_input.testcase
+  setup_input = uworker_input.setup_input
+
   _, testcase_file_path, error = setup.setup_testcase(testcase, job_type,
                                                       setup_input)
   if error:
-    # Because this is trusted, we can trust the error.
-    all_errors = uworker_handle_errors.get_all_handled_errors()
-    error.uworker_input.module_name = __name__
-    uworker_handle_errors.handle(error, all_errors)
-    return None
+    return error
 
   # Initialize variables.
   build_fail_wait = environment.get_value('FAIL_WAIT')
 
+  #TODO(alhijazi): check with @metzman if the blobstore is read-only and
```

Comment:
It's not. We'll need to provide this ahead of time (unless we're handling this already somehow). 

Perhaps we can just inline in the actual stacktrace into `Testcase` in the proto message (if it's actually on the blob store) in some common setup routine for all tasks to make this easier.

WDYT @jonathanmetzman  ?